src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -37,8 +47,7 @@ class DialogFullScreen extends Modal {
    */
   get dialogClasses() {
     return classNames(
-      'carbon-dialog-full-screen__dialog',
-      { 'carbon-dialog-full-screen__dialog--open': this.props.open }
```

Comment:
Is there a reason you got rid of this class? Just that it wasn't being used for anything?